src/components/panel/panel.spec.js

```diff
@@ -1055,10 +1056,55 @@ describe('$mdPanel', function() {
         pressEscape();
 
         expect(myButton).toBeFocused();
+
       });
     });
   });
 
+  describe('postClose logic:', function() {
+      it('postClose when provided', function () {
```

Comment:
These tests should have more descriptive names. See the rest of the `it` tests, they all have the form of `should do something when something...`. These tests are failing in Travis and it wasn't immediately apparent without reading the tests what the test was trying to accomplish.